scripts/manual_operations/tests/test_util.py

```diff
@@ -0,0 +1,42 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+
+"""
+Test cases for util functions
```

Comment:
NITPICK: To be explicit about the purpose of the utils, could we reword this slightly to something along the lines of "Unit tests for manual operations utilities"